### DISCUSSION TITLE
Applying home page via group policy

### DISCUSSION THREAD

**Discussion by dominiceaston** [mozilla/policy-templates#1077] [Q&A]
I'm trying to apply a home page via a group policy, I've set the start page to Homepage and the URL for Home Page but doesn't seem to be applying to the browser. The browser version is 123.0.1. Any help would be appreciated.

**Comment by mkaply** [comment]
Does a home button show up on the toolbar that when clicked goes to the homepage?

Homepage is only seen when the browser is started if you haven't selected to restore your session.

**Reply by dominiceaston** [reply]
No nothing at all it just reverts to the default homepage that comes with Firefox not the homepage I set via group policy. I've applied this same homepage to Edge via the edge template and chrome via the chrome template but doesn't seem to want to work for the Firefox template.

![Screenshot_1](https://github.com/mozilla/policy-templates/assets/162450518/56bb35e8-bb1a-44ee-9034-f8d8b82efa48)

**Comment by mkaply** [comment]
Weird. Can you go to about:policies and see if it has any errors?

**Reply by dominiceaston** [reply]
It says the enterprise policies service is inactive

**Reply by dominiceaston** [reply]
Eventually got it working, it was a problem with the device I was testing on. Thanks for your help.

**Comment by mkaply** [comment]
Awesome!
